README.md

```diff
@@ -111,7 +111,13 @@ a 503.
 In order to be able to securize your virtual host, you have to create a file named as its equivalent VIRTUAL_HOST variable on directory
 /etc/nginx/htpasswd/$VIRTUAL_HOST
 
-    $ docker run -d -p 80:80 -p 443:443 -v /path/to/htpasswd:/etc/nginx/htpasswd -v /path/to/certs:/etc/nginx/certs -v /var/run/docker.sock:/tmp/docker.sock jwilder/nginx-proxy
+```
```

Comment:
hm, no changes for `bash` visible

![bildschirmfoto 2015-04-09 um 22 36 52](https://cloud.githubusercontent.com/assets/649031/7076361/1f1b3c60-df09-11e4-9157-9e9d248c166d.png)

But I'd drop the leading '$' for easier copy & pasting.